api/tests/routes/api/v1/metrics_test.py

```diff
@@ -65,7 +60,7 @@ def test_get_meeting_participants(app, database, session):
             'meeting': meeting1.id,
             'meeting_title': database.specs[0].meeting_subscription.title,
             'participant': user1.email,
-            'time': '04:00PM',
+            'time': '03:00PM',
```

Comment:
This will start to break again when the clocks shift. The proper fix needs timezone awareness which I think was started here but we can go back and forth changing this test when things shift for now: https://github.com/Yelp/beans/pull/61